dart/lib/src/sentry_transaction.dart

```diff
@@ -0,0 +1,47 @@
+import '../sentry.dart';
+import 'utils.dart';
+
+class SentryTransaction extends SentryEvent {
+  late final DateTime _startTimestamp;
+  // move to SentryEvent
+  static const String _type = 'transaction';
+  final List<ISentrySpan> _spans;
+  final Map<String, String> _tags = {};
+
+  SentryTransaction(ISentrySpan trace, this._spans, String name)
+      : super(
+          timestamp: trace.timestamp,
+          transaction: name,
+        ) {
+    _startTimestamp = trace.startTimestamp;
+
+    // tags belong to the event and not trace for transactions
+    final context = trace.context;
+    _tags.addAll(context.tags);
+    context.tags.clear();
```

Comment:
thats just a workaround for now, Java also sets the trace.tags but its wrong

Reply:
tranasction.contexts.trace should not have the tags property, so this is automatically solved